plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -1063,4 +1064,73 @@ function( $key, $hpos_value, $cpt_value ) {
 			array( 'property', 'hpos', 'post' )
 		);
 	}
+
+	/**
+	 * Backfills an order from either the HPOS or the posts datastore.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <id>
+	 * :The ID of the order.
+	 *
+	 * [--from=<hpos|posts>]
```

Comment:
To be coherent with how multiple choice arguments are usually specified in the help of command line tools it could be:

`--from={hpos|posts}`

(angle brackets usually indicate non-literal values that need to be replaced, like in `<order id>`)

And the same with `--to`.

Reply:
This actually got me thinking what the "correct" format should be. I agree with you that the braces seem to better indicate the options, but apparently WP-CLI expects arguments in a specific format (https://make.wordpress.org/cli/handbook/references/documentation-standards/#command-parameter-description) and so I changed the documentation to match that. This actually allows WP-CLI to validate args before the command runs.

I also made the `--from` and `--to` arguments mandatory with no default, to make things more explicit and prevent accidental overwriting of orders.